draft-ietf-dots-signal-channel-26.xml

```diff
@@ -981,7 +981,7 @@
 ]]></artwork>
             </figure></t>
 
-          <t>The corresponding CBOR encoding format is shown in <xref
+          <t>The corresponding CBOR encoding format (for just the payload) is shown in <xref
```

Comment:
This seems like a key point to make, though maybe it is just my unfamiliarity with CoAP speaking.